src/routes/rules/RulesView.vue

```diff
@@ -393,6 +412,7 @@ export default {
   background-color: rgba(var(--v-theme-newPrimary));
   width: 100%;
   padding-top: 16px;
+  padding-bottom: 16px;
```

Comment:
Because this font does not nicely calculate its line heights, applying equal padding to the top and bottom causes the magenta background to appear off-centered:
<img width="829" alt="image" src="https://github.com/cuttle-cards/cuttle/assets/6520704/f15fd061-e18d-49d6-9545-79adbf76a1f0">


Vs 16px top padding with no bottom padding:
<img width="833" alt="image" src="https://github.com/cuttle-cards/cuttle/assets/6520704/c2a5195f-32cb-46bc-b124-7cafc60ebef9">
```suggestion
```

Reply:
This is because, for some annoying reason, "Luckiest Guy" adds 'height' to the text on the bottom, but not the top. you can see this when you highlight the text:
<img width="723" alt="image" src="https://github.com/cuttle-cards/cuttle/assets/6520704/6bbcb00f-d29c-49e1-bf77-d194213923c7">